src/views/ThreeWActivityForm/ActivitiesBySectorInput/ActivityInput/index.tsx

```diff
@@ -0,0 +1,260 @@
+    if (
+        isTruthy(oldValue?.male_count)
+        || isTruthy(oldValue?.female_count)
+    ) {
+        const total = (oldValue.male_count ?? 0)
+            + (oldValue.female_count ?? 0);
+
+        return {
+            ...oldValue,
+            people_count: total,
+        };
+    }
+    return oldValue;
+}
+
+type Props = {
+    clientId: string;
+    mainIndex: number;
+    onChange: (value: SetValueArg<PartialActivityItem>, index: number) => void;
+    error: Error<PartialActivityItem> | undefined;
+    sectorKey: number;
+    value: PartialActivityItem;
+} & ({
+    type: 'custom';
+    handleRemoveClick: (clientId: string) => void;
+    itemNumber: number;
+    actionDetails?: undefined;
+} | {
+    type: 'action';
+    itemNumber?: undefined;
+    actionDetails: Options['actions'][number] | undefined;
+    handleRemoveClick?: undefined;
+});
+
+function ActivityInput(props: Props) {
+    const {
+        clientId,
+        itemNumber,
+        mainIndex,
+        actionDetails,
+        handleRemoveClick,
+        onChange,
+        sectorKey,
+        error: errorFromProps,
+        value,
+        type,
+    } = props;
+
+    const setFieldValue = useFormObject(mainIndex, onChange, {
+        client_id: randomString(),
+        action: actionDetails?.id,
+        sector: sectorKey,
+    });
+
+    const {
+        deployments_emergency_project_activity_people_households: peopleHouseholdOptions,
+    } = useGlobalEnums();
+
+    // FIXME: Write this condition appropriately
+    // This should be shown only if data is not filled
+    // by the user
+    const showNoDataAvailable = true;
+    const error = getErrorObject(errorFromProps);
+    const peopleCountDisabled = isDefined(value?.male_count) || isDefined(value?.female_count);
+    const genderDisaggregationDisabled = isDefined(value?.people_count)
+        && isNotDefined(value?.male_count)
+        && isNotDefined(value?.female_count);
+
+    const handleMaleCountChange = useCallback((newCount: number | undefined) => {
+        onChange((oldValue) => (
+            updateTotalCountChange({
+                ...oldValue,
+                client_id: oldValue?.client_id ?? randomString(),
+                male_count: newCount,
+            })
+        ), mainIndex);
+    }, [onChange, mainIndex]);
+
+    const handleFemaleCountChange = useCallback((newCount: number | undefined) => {
+        onChange((oldValue) => (
+            updateTotalCountChange({
+                ...oldValue,
+                client_id: oldValue?.client_id ?? randomString(),
+                female_count: newCount,
+            })
+        ), mainIndex);
+    }, [onChange, mainIndex]);
+
+    const pointCountInputShown = (
+        (value?.is_simplified_report && type === 'custom')
+        || (
+            value?.is_simplified_report
+            && !actionDetails?.is_cash_type
+            && actionDetails?.has_location
+        )
+    );
```

Comment:
Why check for the same value is_simplified_report twice?

Reply:
To clearly distinguish what happens for custom or action type